src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -14,7 +14,78 @@ void EnKaizoku_Init(Actor* thisx, PlayState* play);
+void func_80B872F4(EnKaizoku* this, PlayState* play);
+void func_80B88214(EnKaizoku* this);
+void func_80B8833C(EnKaizoku* this);
+void func_80B88770(EnKaizoku* this);
+void func_80B88910(EnKaizoku* this);
+void func_80B8760C(EnKaizoku* this, PlayState* play);
+void func_80B8798C(EnKaizoku* this, PlayState* play);
+void func_80B88CD8(EnKaizoku* this);
+void func_80B87D3C(EnKaizoku* this, PlayState* play);
+void func_80B87CF8(PlayState*, Vec3f*);
+void func_80B87E9C(EnKaizoku* this, PlayState* play);
+void func_80B87FDC(EnKaizoku* this, PlayState* play);
+void func_80B88CD8(EnKaizoku* this);
+void func_80B87E28(EnKaizoku* this);
+void func_80B86B58(EnKaizoku* this);
+void func_80B8971C(EnKaizoku* this, PlayState* play);
+void func_80B894C0(EnKaizoku* this, PlayState* play);
+void func_80B89280(EnKaizoku* this, PlayState* play);
+void func_80B88D6C(EnKaizoku* this, PlayState* play);
+void func_80B88964(EnKaizoku* this, PlayState* play);
+void func_80B887AC(EnKaizoku* this, PlayState* play);
+void func_80B88278(EnKaizoku* this, PlayState* play);
+void func_80B88378(EnKaizoku* this, PlayState* play);
+
+u16 D_80B8A8D0[] = {
+    0x11A4, 0x11A5, 0x11A6, 0x11A7, 0x11A8, 0x11A9, 0x11AA, 0x11AB, 0x11AC, 0x11AD,
+};
+
+Vec3f D_80B8A8E4[] = {
+    { -70.0f, 9.0f, 30.0f },   { -70.0f, 9.0f, 30.0f },   { -70.0f, 9.0f, 30.0f },   { 41.0f, 55.0f, 11.0f },
+    { 41.0f, 55.0f, 11.0f },   { 70.0f, 45.0f, 35.0f },   { 80.0f, 45.0f, 25.0f },   { 0.0f, 0.0f, 0.0f },
+    { -70.0f, 9.0f, 30.0f },   { -70.0f, 9.0f, 30.0f },   { -70.0f, 9.0f, 30.0f },   { 41.0f, 55.0f, 11.0f },
+    { 41.0f, 55.0f, 11.0f },   { 70.0f, 45.0f, 25.0f },   { 80.0f, 45.0f, 25.0f },   { 0.0f, 0.0f, 0.0f },
+    { 90.0f, 9.0f, -30.0f },   { 90.0f, 9.0f, -30.0f },   { 90.0f, 9.0f, -30.0f },   { -41.0f, 55.0f, -11.0f },
+    { -41.0f, 55.0f, -11.0f }, { -70.0f, 45.0f, -25.0f }, { -80.0f, 45.0f, -25.0f }, { 0.0f, 0.0f, 0.0f },
+};
+
+Vec3f D_80B8AA04[] = {
+    { 120.0f, 60.0f, -20.0f },   { 120.0f, 60.0f, -20.0f },   { 120.0f, 60.0f, -20.0f },   { -470.0f, 30.0f, -140.0f },
+    { -470.0f, 30.0f, -140.0f }, { -410.0f, 80.0f, -130.0f }, { -410.0f, 80.0f, -130.0f }, { 0.0f, 0.0f, 0.0f },
+    { 120.0f, 60.0f, -20.0f },   { 120.0f, 60.0f, -20.0f },   { 120.0f, 60.0f, -20.0f },   { -470.0f, 30.0f, -140.0f },
+    { -470.0f, 30.0f, -140.0f }, { -410.0f, 80.0f, -130.0f }, { -410.0f, 80.0f, -130.0f }, { 0.0f, 0.0f, 0.0f },
+    { -120.0f, 60.0f, 20.0f },   { -120.0f, 60.0f, 20.0f },   { -120.0f, 60.0f, 20.0f },   { 470.0f, 30.0f, 140.0f },
+    { 470.0f, 30.0f, 140.0f },   { 410.0f, 80.0f, 130.0f },   { 410.0f, 80.0f, 130.0f },   { 0.0f, 0.0f, 0.0f },
+};
+
+Color_RGBA8 D_80B8AB24[] = {
+    { 255, 255, 90, 255 },
+    { 55, 25, 80, 255 },
+    { 5, 195, 110, 255 },
+};
+
+Color_RGBA8 D_80B8AB30[] = {
+    { 255, 130, 10, 255 },
+    { 185, 130, 210, 255 },
+    { 135, 195, 80, 255 },
+};
+
 // static DamageTable sDamageTable = {
 static DamageTable D_80B8AB3C = {
```

Comment:
```suggestion
static DamageTable sDamageTable = {
```
Since data migration has been completed, you can rename these variables with the names given in the comments, and delete the comment